### ISSUE TITLE
Image size not same as container

### ISSUE THREAD

**Issue by SiliconMachine** [zhanziyang/vue-croppa#216]
I have this:

`    <croppa
      :placeholder="'Choose an image'"
      :placeholder-font-size="12"
      :placeholder-color="'black'"
      :width="280"
      :height="280"
      prevent-white-space
      v-model="myCroppa"
      :canvas-color="'transparent'"
    />`

But when I store my image it saves with the double of the height/width although the container has the proper size.

There's something that I'm missing out?

**Comment by RedShift1** [comment]
For some reason the canvas width and size is set to double if the width and height you supply to the component. Seems like a bug to me.

**Comment by Demoskretos** [comment]
Nah, this is just due to poor property naming and default value. The prop you're looking for is **quality**. By default it is set to _2_ which doubles the output size of the image... Just set it to 1 and it should output the same size as the image.

**Comment by ardeleandrei** [comment]
I think this should be quality="1" by default to prevent confusion, I certainly wouldn't have expected it to double the output size of the image when I only need it to crop or reposition an image

**Comment by zhanziyang** [comment]
On vue-croppa 2.x: The `quality` prop (default 2) multiplies the internal canvas resolution so output images remain crisp. CSS viewport size remains governed by `width` and `height`.

Closing as answered. Please feel free to open a new issue if this doesn't cover your use case.
